Library/Homebrew/extend/os/linux/dependency_collector.rb

```diff
@@ -4,7 +4,7 @@ class DependencyCollector
   def java_dep_if_needed(tags)
     req = JavaRequirement.new(tags)
     begin
-      dep = Dependency.new("openjdk", tags)
+      dep = Dependency.new("adoptopenjdk", tags)
```

Comment:
@sjackman This should share a constant with https://github.com/Homebrew/brew/pull/6281 to avoid future confusion.